doc/source/operations/tempest.rst

```diff
@@ -0,0 +1,303 @@
+======================================
+Running Tempest with Kayobe Automation
+======================================
+
+Overview
+========
+
+This document describes how to configure and run `Tempest
+<https://docs.openstack.org/tempest/latest/>`_ using `kayobe-automation
+<https://github.com/stackhpc/kayobe-automation>`_ from the ``.automation``
+submodule included with ``stackhpc-kayobe-config``.
+
+The following guide will assume all commands are run from your
+``kayobe-config`` root and the environment has been configured to run Kayobe
+commands unless stated otherwise.
+
+Building a Kayobe container
+===========================
+``kayobe-automation`` runs in a container on the Ansible control host. This
+means that Docker must be installed on the Ansible control host if it is not
+already.
+
+.. warning::
+
+    Docker can cause networking issues when it is installed. By default it will
+    create a bridge and change ``iptables`` rules. These can be
+    disabled by setting the following in ``/etc/docker/daemon.json``:
+
+    .. code-block:: json
+
+        {
+            "bridge": "none",
+            "iptables": false
+        }
+
+To install Docker on Ubuntu:
```

Comment:
I do wonder if it's time for Kayobe to grow support for configuring the Ansible control host. In the meantime, is it worth wrapping this in a custom playbook that imports the `openstack.kolla.docker` role?

Reply:
Would this only be possible in Zed+? Or should we install it manually?

Reply:
Kayobe installs the kolla collection in yoga, but I don't think it would be in the ANSIBLE_COLLECTIONS_PATH until Zed. Let's keep the manual instructions for now and consider the playbook later.